db_test.go

```diff
@@ -2188,3 +2188,24 @@ func TestUpdateMaxCost(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, int64(4<<20), cost)
 }
+
+func TestOpenDB(t *testing.T) {
+	dir, err := ioutil.TempDir("", "badger-test")
+	require.NoError(t, err)
+	defer os.RemoveAll(dir)
+
+	ops := getTestOptions(dir)
+	ops.ReadOnly = false
+	db, err := Open(ops)
```

Comment:
Write some data which goes to sst and vlog file.

Reply:
Done.